lib/wasi/src/state/mod.rs

```diff
@@ -1196,7 +1239,18 @@ impl WasiFs {
         is_preopened: bool,
         name: String,
     ) -> Inode {
-        let mut stat = __wasi_filestat_t::default();
+        let stat = __wasi_filestat_t::default();
+        self.create_inode_with_stat(kind, is_preopened, name, stat)
+    }
+
+    /// creates an inode with the given filestat and insert it.
```

Comment:
```suggestion
    /// Creates an inode with the given filestat and insert it.
```
Rule of thumb is that either it's a sentence which starts with a capital and ends with a period (or question or exclamation mark) or it's a sentence fragment and has neither.